package.json

```diff
@@ -44,7 +44,6 @@
     "sinon": "^19.0.2"
   },
   "dependencies": {
-    "@formatjs/intl-pluralrules": "^1",
```

Comment:
Just to clarify: the other things that this polyfill was providing (the plural rules locale data for browsers who don't support the native API) aren't needed at all anymore?

Reply:
Ack! Safari 12 is the only browser that isn't blocked that doesn't support `Intl.Pluralules`. Do we want to keep it for that?

Reply:
Discussed offline.

Brightspace loads... something in Safari 12 but it's completely unuseable so we're going ahead with removing the polyfill. We expect to block Safari 12 in the near future.